GeoJSON/geojson-encoding-rule.md

```diff
@@ -0,0 +1,270 @@
+
+The GeoJSON format cannot deal with 3D geometries and coverage/raster data, therefore this encoding rule cannot be used for those types of data.
+
+### INSPIRE Requirements for Encoding Rules
+
+The Implementing Rules on interoperability of spatial data sets and services (Commission Regulation (EU) No 1089/2010) lays down the following requirements for encoding rules:
+
+> **Article 7 -- Encoding**
+>
+> 1. Every encoding rule used to encode spatial data shall conform to EN ISO 19118. In particular, it shall specify schema
+> conversion rules for all spatial object types and all attributes and association roles and the output data structure used. 
+> 2. Every encoding rule used to encode spatial data shall be made available.
+
+D2.7 specifies more detailed requirements and recommendations for encoding rules. The following list lists the requirements from that document and shows which ones are also met in this encoding rule:
+
+> * Requirement 12: ... documents shall be required to be encoded using UTF-8 as character encoding.
+
+D2.7 also contains a relevant recommendation:
+
+> * Recommendation 3: Encoding rules should be based on international, preferably open, standards.
+
+## Normative References
+
+This section contains references to standards documents and related resources.
+
+* [GeoJSON - IETF RFC 7946](https://tools.ietf.org/html/rfc7946)
+* [D2.7 INSPIRE DRAFTING TEAM “DATA SPECIFICATIONS.” D2.7: Guidelines for the encoding of spatial data, Version 3.3](http://inspire.ec.europa.eu/documents/guidelines-encoding-spatial-data)
+
+## Terms and Definitions
+
+Terms and Definitions can be found in the [Glossary](../glossary.md) document.
+
+## Schema Conversion Rules
+
+INSPIRE defines the conceptual model using UML.
+
+The default encoding rule maps this UML model to XML Schema. For JSON, [JSON Schema](https://json-schema.org/) can be used to perform simple validation on JSON documents.  
+
+NOTE At this point, [JSON Schema](https://json-schema.org/) is not used in any of the targeted GIS tools. We thus do not yet use it normatively, but to provide a formal definition of the encoded data structures, and to help develop executable test suites in future versions. 
+
+In this encoding rule, we take a two-step approach, where we apply model transformations on the level of the conceptual model. This model can then be encoded in simple GeoJSON using the general schema and instance conversion rules laid out in the next sections.
+
+### Types
+
+#### Feature types 
+
+All types that have the stereotype `<<featureType>>` are converted to GeoJSON objects. Typenames remain as they are. 
+
+#### Data Types
+
+All data types are mapped to regular JSON objects, where properties of the type are directly added to the root object instead of being added to the `properties` subobject. Their typenames remain as they are.
```

Comment:
I would remove the "instead of in the 'properties' subobject" here and rather add that for feature types above.